store/sqlstore/post_store.go

```diff
@@ -322,7 +322,8 @@ type etagPosts struct {
 
 func (s SqlPostStore) InvalidateLastPostTimeCache(channelId string) {
 	lastPostTimeCache.Remove(channelId)
-	lastPostsCache.Remove(channelId)
+	lastPostsCache.Remove(channelId + "30")
```

Comment:
Is there any way to document the `30` and `60` here? Without context, they seem "magical".

Reply:
I can add comments but the fact is they are pretty magical :P